### DISCUSSION TITLE
give permission for fork

### DISCUSSION THREAD

**Discussion by Akash-Gupta-git** [sanjay-kv/Open-source-Practice#1121] [General]
Hello sir @sanjay-kv 
I want to give permission for fork my repo when someone access my repo
like  as you shared a website in the video 
video name  'make custom readme page' there you show someone's profile and it ask fork

**Comment by sanjay-kv** [comment]
go to settings and you can enable or disable it

**Reply by Akash-Gupta-git** [reply]
oh yeh!

**Comment by abhisek2004** [comment]
### How to Allow Forking of Your Repository

1. **Make Your Repo Public** 🌍:
   - Ensure your repository is public in the **Settings** tab. Private repositories can’t be forked unless users are collaborators.

2. **Encourage Forking in README** 📄:
   - Add a friendly note in your README.md:
     ```markdown
     ## 🤝 Fork This Repository!
     If you find this project useful, feel free to fork it and contribute!
     ```

3. **Promote Your Repo** 📢:
   - Share your repository link on social media and forums, encouraging others to fork and contribute.

### Important Note ⚠️:
- Always follow GitHub’s guidelines to avoid any issues with your account. Avoid spamming and ensure that your content complies with their community standards.

By following these steps, you’ll make it easy for others to fork your repository while keeping your account safe!

If you have more questions, feel free to ask! 😊

**Reply by Akash-Gupta-git** [reply]
okay
